src/vtp/web/api/main.py

```diff
@@ -1,19 +1,38 @@
 """API endpoints for the VoteTrackerPlus backend"""
-import json
-import random
 
-from fastapi import FastAPI
-from importlib_resources import files
+# import json
+import os
 
-import vtp.web.api.data
+from fastapi import FastAPI
+from vtp.ops.accept_ballot_operation import AcceptBallotOperation
+from vtp.ops.cast_ballot_operation import CastBallotOperation
+from vtp.ops.setup_vtp_demo_operation import SetupVtpDemoOperation
 
 app = FastAPI()
 
-# create a list to store VoteStoreIDs
+########
+# backend demo constants
+########
+# where the ElectionData repo is
+DEFAULT_ELECTION_DATA_DIR = (
+    "/opt/VoteTrackerPlus/demo.01/mock-clients/scanner.00/VTP-mock-election.US.13"
+)
+# pylint: disable=line-too-long
+DEFAULT_BLANK_BALLOT = "GGOs/states/Massachusetts/GGOs/towns/Concord/blank-ballots/json/000,001,002,003,ballot.json"
+# for testing only
```

Comment:
I think it will go away first as the backend probably wants to deal with it.  The reason it is here is that the client needs the backend to invoke some git commands before there is a client allocated workspace in which to run git commands.  This can go away in several ways